testutils/trace_context.go

```diff
@@ -7,43 +7,48 @@ import (
 // SimpleTraceContextSource is a dummy implementation of TraceContextSource.
 type SimpleTraceContextSource struct{}
 
-// NewRootTraceContext imlements NewRootTraceContext of opentracing.TraceContextSource.
+// NewRootTraceContext implements NewRootTraceContext of opentracing.TraceContextSource.
 func (source *SimpleTraceContextSource) NewRootTraceContext() opentracing.TraceContext {
 	return nil
```

Comment:
Hard to say. I was sad about the (span, tags) thing for NewChild but felt like I was painted into a corner by the motivating use case. I didn't have such a motivating real-world use case for this one, though I don't like the smell of the asymmetry.